test/OpenTelemetry.Tests/Logs/LoggerProviderBuilderExtensionsTests.cs

```diff
@@ -154,11 +169,15 @@ public void LoggerProviderBuilderAddProcessorTest()
                 processors.Add(processor);
                 Assert.False(processor.Disposed);
 
+                Assert.True((int)processor.PipelineWeight >= lastWeight);
```

Comment:
I think we could make this test more readable. Could we add a name field to `CustomProcessor`? We could then name the `CustomProcessor` instances and check if they were added in order. Something like add five instances of `CustomProcessor` : A, B, C, D, and E with different weights and expect them to be added in the right order such as B, C, A, D, E.




```

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/5424/commits/62a9f40cdb55debe9a2aa6e42fe6c1f50e3faf67